src/NUnitFramework/framework/Api/FrameworkController.cs

```diff
@@ -114,17 +114,19 @@ private void Initialize(string assemblyNameOrPath, IDictionary settings)
             var newSettings = settings as IDictionary<string, object>;
             Settings = newSettings ?? settings.Cast<DictionaryEntry>().ToDictionary(de => (string)de.Key, de => de.Value);
 
-            if (Settings.ContainsKey(FrameworkPackageSettings.InternalTraceLevel))
+            if (Settings.TryGetValue(FrameworkPackageSettings.InternalTraceLevel, out var traceLevelValue))
             {
-                var traceLevel = (InternalTraceLevel)Enum.Parse(typeof(InternalTraceLevel), (string)Settings[FrameworkPackageSettings.InternalTraceLevel], true);
+                var traceLevel = (InternalTraceLevel)Enum.Parse(typeof(InternalTraceLevel), (string)traceLevelValue, true);
 
-                if (Settings.ContainsKey(FrameworkPackageSettings.InternalTraceWriter))
-                    InternalTrace.Initialize((TextWriter)Settings[FrameworkPackageSettings.InternalTraceWriter], traceLevel);
+                if (Settings.TryGetValue(FrameworkPackageSettings.InternalTraceWriter, out var textWriterValue))
+                {
+                    InternalTrace.Initialize((TextWriter)textWriterValue, traceLevel);
+                }
                 else
                 {
-                    var workDirectory = Settings.ContainsKey(FrameworkPackageSettings.WorkDirectory)
-                        ? (string)Settings[FrameworkPackageSettings.WorkDirectory]
-                        : Directory.GetCurrentDirectory();
+                    var workDirectory = Settings.TryGetValue(FrameworkPackageSettings.WorkDirectory, out var workDirectoryValue) ?
+                        (string)workDirectoryValue :
+                        Directory.GetCurrentDirectory();
```

Comment:
It's harder for me to scan code when the operators aren't at the start of the line. Was this part of the change intentional?

Reply:
nope. will move to start

Reply:
Can you move the `:` back too?